src/api/language.js

```diff
@@ -24,6 +25,20 @@ const loadMessages = (lang) => {
     .then((messages) => Object.assign.apply({}, messages))
 }
 
+const loadAllLanguagesMessages = () => {
+  const messages = {}
+  allLanguages.forEach(language => {
+    try {
+      messages[language.lang] = require(`../../i18n/${language.lang}.json`)
+    } catch (error) {
+      // Translation not available, discard
+    }
+  })
+
+  return messages
+}
+
 export {
-  loadMessages
+  loadMessages,
+  loadAllLanguagesMessages
 }
```

Comment:
Once we have `loadAllLanguagesMessages` in this file, we are safe to move this under `utils`, as this becomes a method not related to `api`

Reply:
Sounds good, how about including it directly under`src/utils/languages.js`: 00e08c4b2c1c79c953af456d787609344a443491